news/feature-3754.md

```diff
@@ -0,0 +1,9 @@
+`disk-buffer`: Added a fast-path memory queue implementation to the reliable disk-buffer,
+which can save time, when messages are coming in with a smaller rate, similarly to the
+qout memory queue in the non-reliable disk-buffer.
+
+This queue still guarantees, that all the received messages are written to the disk
+as soon as possible.
+
+Also added `qout-size()` option to reliable disk-buffer, which limits the number of
+messages in the new memory queue.
```

Comment:
I think we should be more concise, so we don't confuse users about queue segments in the disk buffer.

One thing I would mention is that we added a new **option** to the reliable disk buffer: `qout-size()`, which defines the number of messages that are kept in memory **in addition to writing it to disk**. This serves performance purposes and offers the same no-message-loss guarantees as before. It can be used to maintain a higher throughput when only a small number of messages are waiting in the disk buffer.

What do you think?

Reply:
That is a much better articulation of this feature, thanks. I have modified the news file accordingly.